docs/source/api-docs/glowgr-functions.rst

```diff
@@ -0,0 +1,8 @@
+GloWGR
+------
+
+.. automodule:: glow.wgr.linear_model.ridge_model
```

Comment:
I think the ordering might make more sense if we put the general functions on top, followed by the linear-model specific ones below. It might also help to have a little exposition for each class.

Reply:
Did you take a look at the rendered documentation? We include the docstring, which is pretty thorough.

Reply:
If more exposition is required, it should go in the docstring.

Reply:
I was thinking more in terms of explicitly denoting the different subsections - the first section is for general use, the second section is for the linear model, and the last section will be for the binary model. But we can clean that up when the binary model goes in.

Reply:
Ah, so just like section headers? I'll add that.